### DISCUSSION TITLE
running mm-snap fails

### DISCUSSION THREAD

**Discussion by elenamik** [MetaMask/snaps#696] [Q&A] [answered]
Hi,

I was running the `mm-snap` cli without a hitch, and after I tried to add TS to my snap project I started getting the below error when attempting to run any mm-snap commands, and now can't run mm-snap at all anymore. I have uninstalled + re-installed globally many times but the result is the same. I dont know if it was caused by me adding TS to my repo, but the error looks like it might be typescript related, so its a hypothesis.

```
$ mm-snap
node:internal/modules/cjs/loader:959
  throw err;
  ^

Error: Cannot find module './snap-config.__GENERATED__'
Require stack:
- /Users/Lena/.config/yarn/global/node_modules/@metamask/snaps-cli/dist/utils/snap-config.js
- /Users/Lena/.config/yarn/global/node_modules/@metamask/snaps-cli/dist/utils/index.js
- /Users/Lena/.config/yarn/global/node_modules/@metamask/snaps-cli/dist/cli.js
- /Users/Lena/.config/yarn/global/node_modules/@metamask/snaps-cli/dist/main.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:956:15)
    at Function.Module._load (node:internal/modules/cjs/loader:804:27)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/Lena/.config/yarn/global/node_modules/@metamask/snaps-cli/dist/utils/snap-config.js:12:36)
    at Module._compile (node:internal/modules/cjs/loader:1126:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1180:10)
    at Module.load (node:internal/modules/cjs/loader:1004:32)
    at Function.Module._load (node:internal/modules/cjs/loader:839:12)
    at Module.require (node:internal/modules/cjs/loader:1028:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/Lena/.config/yarn/global/node_modules/@metamask/snaps-cli/dist/utils/snap-config.js',
    '/Users/Lena/.config/yarn/global/node_modules/@metamask/snaps-cli/dist/utils/index.js',
    '/Users/Lena/.config/yarn/global/node_modules/@metamask/snaps-cli/dist/cli.js',
    '/Users/Lena/.config/yarn/global/node_modules/@metamask/snaps-cli/dist/main.js'
  ]
```

I have confirmed that the `snap-config.__GENERATED__` indeed does not exist, the only file under `snap-config.js` is `snap-config.js.map`

<img width="1052" alt="Screen Shot 2022-08-19 at 10 19 11 AM" src="https://user-images.githubusercontent.com/44687983/185638961-9b27aae6-eeee-4f00-b4d6-242fdddbc2b7.png">

Has anyone come across this before with this CLI? I am not sure if this is an issue with my computer and how yarn is compiling files, or if it's an issue with the CLI. As far as I know, all other yarn deps are working fine (nextjs, etc).

Would appreciate the help! Thanks

**Comment by ritave** [comment, marked as answer]
Hello @elenamik, the `0.19.0` release had a bug in `mm-snaps` that crashed cli at startup.

We've just released `0.19.1` that fixes the problem.

**Reply by elenamik** [reply]
Ahhh thank you, I was racking my brain trying to figure out what was wrong. Appreciate the quick response!
